readme.md

```diff
@@ -0,0 +1,67 @@
+# cvsclone
+
+Utility to clone cvs repositories over the cvspserver interface.
+
+As ```git cvsimport``` is pretty slow over the wire (and as [cvs2git](http://cvs2svn.tigris.org/cvs2git.html) can do a substantially better job if there are a lot of branches), it is often better to "clone" a whole CVS repository. Enter cvsclone!
+
+## Features
+
+- Works with anonymous access. Will read $HOME/.cvspass if needed
+- Can clone corrupt repositories: writes ,v files directly, does not need rcs (For example, ccvs module has archives that go backwards in time).
+
+## Building
+
+You will need [Flex](http://flex.sourceforge.net/) installed to create cvsclone. You can normally find this in your package manager i.e:
+
+```bash
+> apt-get install flex
+```
+
+Use [Homebrew](http://brew.sh/) on OS X:
+```bash
+> brew install flex
+```
+
+Once installed, clone this repository and build using **make**:
+
+```bash
+> git clone

```

Comment:
`git clone https://github.com/akavel/cvsclone` no? 

also, as far as I've seen, it seems less unusual to use `$` than `>` as prompt marker in bash snippets